addons/binding/org.openhab.binding.omnikinverter/src/main/java/org/openhab/binding/omnikinverter/OmnikInverter.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```

Comment:
Please use the new EPL-v2.0 header. See also this [community topic](https://community.openhab.org/t/tutorial-update-license-to-epl-v2-0-of-your-binding-in-development-or-pull-request/65088?u=wborn).